### ISSUE TITLE
MethodData.loadMethod returns null on found method

### ISSUE THREAD

**Issue by Mino260806** [NeonOrbit/Dexplore#6]
### Method Signature
```
com.facebook.messaging.msys.common.translator.AbstractMsysMessagesCollectionTranslator.A00(com.facebook.messaging.model.threadkey.ThreadKey,com.facebook.messaging.msys.common.translator.AbstractMsysMessagesCollectionTranslator,com.facebook.messengerorcacqljava.OrcaTempMessageList,X.5JD,com.facebook.xapp.messaging.capability.vector.Capabilities,com.google.common.collect.ImmutableSet,java.lang.Number,java.util.List,java.util.Map,int,int):com.facebook.messaging.model.messages.MessagesCollection
```
### App
Messenger version v416 ([download link](https://github.com/Mino260806/MessengerPro/releases/download/v1.0.5/messenger416-pie.apk))

**Comment by NeonOrbit** [comment]
@Mino260806 Please make sure you are using the latest version, as this bug was fixed a while ago.

**Comment by Mino260806** [comment]
Solved by updating to version 1.4.5
